routes/index.js

```diff
@@ -1,16 +1,62 @@
 const express = require('express');
+const { errors, celebrate, Joi } = require('celebrate');
 const users = require('./users');
 const cards = require('./cards');
 const auth = require('../middlewares/auth');
 const NotFoundError = require('../errors/not-found-err');
+const { createUser, login } = require('../controllers/users');
+
+const { requestLogger, errorLogger } = require('../middlewares/logger');
 
 const router = express.Router();
 
 router
-  .use('/cards', auth, cards)
-  .use('/users', auth, users)
+  .use(requestLogger)
+  .post('/signin', celebrate({
+    body: Joi.object().keys({
+      email: Joi.string().required().email(),
+      password: Joi.string().required().min(8),
+    }),
+  }), login)
+  .post('/signup', celebrate({
+    body: Joi.object().keys({
+      name: Joi.string().required().min(2).max(30),
+      about: Joi.string().required().min(2).max(30),
+      avatar: Joi.string().required().uri(),
+      email: Joi.string().required().email(),
+      password: Joi.string().required().min(8),
+    }),
+  }), createUser)
+  .use('/cards', celebrate({
+    cookies: Joi.object().keys({
+      jwt: Joi.string(),
+    }).unknown(true),
+  }), auth, cards)
+  .use('/users', celebrate({
+    cookies: Joi.object().keys({
+      jwt: Joi.string(),
+    }).unknown(true),
+  }), auth, users)
+  .get('/crash-test', (req, res, next) => {
+    setTimeout(() => {
+      next(new Error('Сервер сейчас упадёт'));
+    }, 0);
+  })
```

Comment:
Нужно исправить: при обращении на роут краш теста сервер должен падать.
Особенность падения в том, что ошибку не нужно обрабатывать. Этот роут нужен лишь для того, чтобы научиться восстанавливать работоспособность. Поэтому верните как было:
```suggestion
  .get('/crash-test', (req, res, next) => {
    setTimeout(() => {
      throw new Error('Сервер сейчас упадёт');
    }, 0);
  })
```

Дополнительное здесь используется `setTimeout()`, чтобы ошибка не была поймана обработчиком ошибок.
Ожидаемый результат при обращении на роут - сервер падает и приходит стандартный ответ Nginx с кодом 502.
После обращения на другой роут приходит ответ такой, который ожидается на этом роуте.

Reply:
исправлено